app/uk/gov/hmrc/cgtpropertydisposalsfrontend/views/returns/exemptionandlosses/further_return_in_year_losses.scala.html

```diff
@@ -129,13 +127,6 @@
             customErrorKey = Some(s"$key$userKey")
         )
 
-        <p>@Html(
-            messages(
-                s"$key$furtherReturnKey$userKey.link",
-                triageRoutes.FurtherReturnGuidanceController.guidance(FurtherReturnGuidanceController.BackLinkLocations.inYearLosses)
```

Comment:
the messages for this link can be deleted now. They're under `inYearLosses.furtherReturn.link` with the user variations below it. We also don't need `FurtherReturnGuidanceController.BackLinkLocations.inYearLosses` anymore